app/src/main/java/edu/mills/cs115/fruitthief/database/PopulateFruitTable.kt

```diff
@@ -0,0 +1,37 @@
+package edu.mills.cs115.fruitthief.database
+
+import kotlinx.coroutines.runBlocking
+import timber.log.Timber
+
+class PopulateFruitTable(fruitTreeDao: FruitTreeDAO) {
+
+    init {
+        runBlocking{
+            if (fruitTreeDao.getFruitList().isEmpty()) {
+                Timber.i("Populating table")
+                fruitTreeDao.insert(Fruit(0, "Apple", "HIJK"))
+                fruitTreeDao.insert(Fruit(0, "Apricot", "FG"))
+                fruitTreeDao.insert(Fruit(0, "Asian Pear", "IJK"))
```

Comment:
The fruit names should all be extracted into `strings.xml` (but not the seasons).